### PR TITLE
Remove doctor/nurse as search results aren't good enough

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

File: src/pages/personalisation/HealthSubcategories.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* @flow */
 
-import BaseQuestion, { remove, append } from "./BaseQuestion";
+import BaseQuestion, { remove } from "./BaseQuestion";
```
Comment (jscn) [inline src/pages/personalisation/HealthSubcategories.js:3]: This isn't the same `append` being used on L13?

Comment (DanielHeath) [inline src/pages/personalisation/HealthSubcategories.js:3]: no, thats a method call

Comment (jscn) [inline src/pages/personalisation/HealthSubcategories.js:3]: Cool, tired eyes. Looks good.

**Comment by jscn** [comment]
++, let's get it checked in staging before we tag a release

**Pull request merged**